src/ClientData/include/ClientData/FunctionUtils.h

```diff
@@ -20,7 +20,7 @@ namespace orbit_client_data::function_utils {
 
 [[nodiscard]] inline const std::string& GetDisplayName(
```

Comment:
We should consider removing this function now.

Reply:
I like that it centralizes the logic to get the string to display, it might be handy if ever we need modify the way we display functions in the data views. I'd prefer leaving as is.